style.css

```diff
@@ -1354,4 +1383,5 @@ h1.big-heading {
 
 .container-faq {
   padding: 3%;
+}
```

Comment:
seems extra

Reply:
Recheck and confirm once please

Reply:
Checked it, it was set to maintain for all pixels, but yes it can work without that too.

I have committed with the required changes, please check it once.